site/components/Navigation/NavItemMobile.tsx

```diff
@@ -1,26 +1,39 @@
-import Link from "next/link";
-import React, { FC } from "react";
+import React, { Dispatch, FC, SetStateAction, ReactElement } from "react";
 import * as styles from "./styles";
-
+import { cx } from "@emotion/css";
+import { ButtonPrimary } from "@klimadao/lib/components";
+import { NavItem } from "./index";
 interface Props extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
   name: string;
+  id: NavItem;
   url: string;
-  number?: string;
+  selected: string | undefined;
+  setSelected: Dispatch<SetStateAction<NavItem | undefined>>;
+  buttons?: ReactElement<any, any>[];
```

Comment:
Same here: Correct type is  `JSX.Element[];`

Reply:
type updated